app/controllers/proposals_controller.rb

```diff
@@ -122,11 +122,12 @@ def require_speaker
   end
 
   def setup_flash_message
-    message = "Thank you for submitting a proposal."
+    message = "Thank you! Your proposal has been submitted and may be reviewed at any time while the CFP is open.\n
```

Comment:
So there's an odd thing with spaces and p tags happening here.  My gut says make each line it's own string with two \n characters at the end.

Also, use the << operator to add to message for each line as you go such like this:

message = "Thank you...\n\n"  (use = for the first time to assign to message)
message << "You are welcome to...\n\n"
...
message << "Expect a response..."

See the chrome dev tool screenshot:
![flash_message_spacing](https://cloud.githubusercontent.com/assets/2417/17754072/162b1a54-6490-11e6-8aa2-9faf641fd725.png)